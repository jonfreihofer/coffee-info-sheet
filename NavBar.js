import React from "react"



function NavBar(){
    return (
        <nav className="navbar">
        <h1>C<span>o</span>ffee Inf<span>o</span> Sheet</h1>
        <button className="nav-toggle">
        <span class="hamburger"></span>
        </button>
        </nav>
    )
}


export default NavBar