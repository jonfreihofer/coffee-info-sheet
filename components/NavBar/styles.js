import styled from "styled-components";

/* nav stuff */
export const StyledNav = styled.nav`
  display: flex;
  justify-content: center;
  max-width: 100%;
  overflow: hidden;
  background-color: #333;
  top: 0;
  width: 100%;
  padding: 1.5em;
  background: #1c1c1c;

  h1 {
    text-transform: uppercase;
    letter-spacing: 2px;
    font-size: 2rem;
    color: white;
    cursor: pointer;

    /* was trying to make the o's look like coffee beans...haha */
    span {
      color: orangered;
    }
  }

  .nav-toggle {
    border: 0;
    background: transparent;
    padding: 0.25rem;
  }

`;
