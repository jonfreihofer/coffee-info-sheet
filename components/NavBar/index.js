import React from "react";
import {StyledNav} from "./styles";

function NavBar() {
  return (
    <StyledNav>
      <h1>
        C<span>o</span>ffee Inf<span>o</span> Sheet
      </h1>
      <button className="nav-toggle">
        <span className="hamburger"></span>
      </button>
    </StyledNav>
  );
}

export default NavBar;
