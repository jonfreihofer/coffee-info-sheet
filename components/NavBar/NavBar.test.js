import React from "react";
import { render } from "@testing-library/react";
import NavBar from ".";

describe("NavBar", () => {
  it("Should render", () => {
    const { findByText, asFragment } = render(<NavBar />);

    expect(findByText("Coffee Info Sheet")).toBeTruthy();
    expect(asFragment()).toMatchSnapshot();
  });
});
