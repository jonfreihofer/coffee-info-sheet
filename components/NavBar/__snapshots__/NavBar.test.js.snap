// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Should render 1`] = `
<DocumentFragment>
  <nav
    class="sc-bdfBwQ FQehq"
  >
    <h1>
      C
      <span>
        o
      </span>
      ffee Inf
      <span>
        o
      </span>
       Sheet
    </h1>
    <button
      class="nav-toggle"
    >
      <span
        class="hamburger"
      />
    </button>
  </nav>
</DocumentFragment>
`;
